contracts/staking/IStakingBase.sol

```diff
@@ -93,12 +93,6 @@ interface IStakingBase is IStakingData {
         uint32 cooldownBlocks
     );
 
-    /**
-     * @dev Emitted when `caller` set `assetHolder` address as `allowed` to send funds
-     * to staking contract.
-     */
-    event AssetHolderUpdate(address indexed caller, address indexed assetHolder, bool allowed);
```

Comment:
(GitHub won't let me comment on the correct LOC as you did not modify it but...)

The event `RebateCollected` uses `assetHolder` name for the query fees provider. We probably want to change that name for clarity, something like `rebateProvider` or `provider`, `sender`, wdyt?